src/libs/SidebarUtils.js

```diff
@@ -216,9 +216,8 @@ function getOptionData(reportID) {
         isPolicyExpenseChat: false,
     };
 
-    const personalDetailMap = OptionsListUtils.getPersonalDetailsForLogins(report.participants, personalDetails);
-    const personalDetailList = _.values(personalDetailMap);
-    const personalDetail = personalDetailList[0] || {};
+    const participantPersonalDetailList = _.values(OptionsListUtils.getPersonalDetailsForLogins(report.participants, personalDetails));
```

Comment:
The `participantPersonalDetailList` gets sent to `ReportUtils.getDisplayNamesWithTooltips` [here](https://github.com/Expensify/App/pull/14008/files#diff-164819655552b8656799f246055951a2c8af7509d3b15063fddd328e358299bbR243) so that we can display the group of people in the chat by their names. It also gets used in `OptionsListUtils.getSearchText` [here](https://github.com/Expensify/App/pull/14008/files#diff-164819655552b8656799f246055951a2c8af7509d3b15063fddd328e358299bbR300) so that we can search for this specific chat by the participant's names.

It's essentially a subset that's important to have in other contexts.